config/variables/dev.hcl

```diff
@@ -1,8 +1,11 @@
 locals {
-  api_timeout                                = 10000
-  environment                                = "dev"
-  common_vnet_address_space                  = "10.1.0.0/16"
-  google_analytics_id                        = "G-X21W2S2FN3"
+  api_timeout                     = 10000
+  appeals_service_public_url      = "appeals-service-dev.planninginspectorate.gov.uk"
```

Comment:
This looks like an appeals service specific variable, so you can put this in the stack variables files I think!

Reply:
Same for applications service variables

Reply:
These are used by the Front Door stack to know what to listen for, as well as the app stacks as discussed.